modules/administration/pages/tuning-changelogs.adoc

```diff
@@ -12,5 +12,25 @@ The parameter defaults to [systemitem]``0``, which means unlimited.
 java.max_changelog_entries = 0
 ----
 
-When you set this parameter, it will come into effect only when packages are synchronized.
+When you set this parameter, it will come into effect only for new packages as they are synchronized.
 You might like to delete the cached data to remove older data.
+You can delete cached data from the command line, using the SQL interface:
+
+[IMPORTANT]
+====
+This process can take a long time.
+Depending on the number of channels you have and the amount of data to be deleted, it can potentially take several hours.
+The task is run in the background by Taskomatic, so you can continue to use {productname} while the operation completes, however you should expect some performance loss.
+====
+
+----
+spacewalk-sql -i
```

Comment:
@mcalmer do we want to suggest use of direct SQL in our manuals?

Reply:
It is the only way to force the re-generation which is needed.

The only other way would be if we write a script which does it and you can call it.
Would be an option.

Reply:
I'm personally not against directly using SQL, as long as it's the simplest method to achieve the goal. If an easier method becomes available, we can update :)

Reply:
thanks for all your reviews!